broadlink/__init__.py

```diff
@@ -1,95 +1,93 @@
 #!/usr/bin/python3
 """The python-broadlink library."""
 import socket
-from typing import Dict, List, Union, Tuple, Type
+from typing import Generator, List, Union, Tuple
 
 from .alarm import S1C
 from .climate import hysen
 from .cover import dooya
 from .device import device, scan
 from .exceptions import exception
 from .light import lb1
-from .remote import rm, rm2, rm4
+from .remote import rm, rm4
 from .sensor import a1
 from .switch import bg1, mp1, sp1, sp2, sp4
 
 
-def get_devices() -> Dict[int, Tuple[Type[device], str, str]]:
```

Comment:
Changing this class to a dictionary is not mandatory, but I think it is much better because the creation of the dictionary is the most complex task. We should do it only once, and access it later.